### ISSUE TITLE
[3.3.1] Doesn't start after update from 3.3.0 to 3.3.1

### ISSUE THREAD

**Issue by alwashe** [Ultimaker/Cura#3732]
**Application Version**
Update from 3.3.0 to 3.3.1

**Platform**
Windows 10 - i7 - 32GB 

**Steps to Reproduce**
update from 3.3.0 to 3.3.1

**Actual Results**
Windows 10 freezes after installation of 3.3.1
after Restart Cura 3.3.1 doen't start, Edit: shortcut broken
Configs: [cura.zip](https://github.com/Ultimaker/Cura/files/1950206/cura.zip)

Edit: After starting directly:
[err_traceback.txt](https://github.com/Ultimaker/Cura/files/1950236/err_traceback.txt)
[logs.txt](https://github.com/Ultimaker/Cura/files/1950237/logs.txt)

**Comment by alwashe** [comment]
after uninstalling, deleting configs and reinstalling works again

**Comment by ianpaschal** [comment]
It appears some of your configs became corrupted with invalid data. I'm glad it's working now, we'll have to see how we can avoid it happening again in the future.

**Comment by PreownedFIN** [comment]
I also had a similar problem. The loading screen popped up and it crashed when trying to load my own quality presets.

I didn't save the error message nor the log as the dialog said they will be sent automatically.

The problem went away after I clicked the "backup and reset configs"-button.
It didn't mention anything where the configs would be backed up. So where are they backed up to?

I upgraded from 3.2.0.
